.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- TODO: Replace the title below -->
+# Your Title
+
+## Description
+
```

Comment:
This change is already included in #25, let's keep in in one place to avoid conflicts

Reply:
Thanks Noor! I just removed the file.